toytools/collect.py

```diff
@@ -120,23 +119,29 @@ def load_image(root : str, is_fake : bool, name : str) -> np.ndarray:
     with np.load(path) as f:
         return f[f.files[0]]
 
-def train_test_split(
+def train_val_test_split(
     n         : int,
+    val_size  : Union[int, float],
     test_size : Union[int, float],
     shuffle   : bool,
     prg       : np.random.Generator,
-) -> Tuple[np.ndarray, np.ndarray]:
-    """Split dataset of size `n` into training/test parts.
+) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+    """Split dataset of size `n` into training/val/test parts.
 
     Parameters
     ----------
     n : int
         Size of the dataset to split.
+    val_size : int or float
+        Fraction of the dataset that will be used as a val sample.
+        If `test_size` <= 1, then it is treated as a fraction, i.e.
```

Comment:
`val_size`?

Reply:
Whoops. Fixed in: [9a7d6e1](https://github.com/LS4GAN/toytools/pull/19/commits/9a7d6e1b3a6f985f3fad42dbe7e545b8e989c858). Thanks!